mplc/partner.py

```diff
@@ -40,4 +40,51 @@ def corrupt_labels(self):
 
     def shuffle_labels(self):
         for label in self.y_train:
-            label = shuffle(label)
+            shuffle(label)
+
+
+class PartnerMpl:
```

Comment:
Any chance to rename this one?

Reply:
Yes please ! I am not satisfied either, but I was not able to find a more suitable name. Any suggestion ?

Reply:
`PartnerInMlp`...? I'm not super inspired...